app/views/staff/index.html.erb

```diff
@@ -0,0 +1,81 @@
+
+<section id ="page-contents" >
+
+  <%= form_for 'staff/training_session', :url => { :action => "create", controller: 'staff/training_sessions'} do |f| %>
+
+    <table class="staff-table">
+      <tr>
+        <th>Select</th>
+        <th>Username</th>
+        <th>Last Signed In</th>
+        <th>Location</th>
+        <th>Certification</th>
+      </tr>
+
+    <% @users.each do |user| %>
+      <tr>
+
+        <td class="checkbox-cell">
+          <input type="checkbox" class="user-checkbox" onclick="check_users(this)"></input>
+        </td>
+
+        <td class="username-cell"><%=
+          link_to(content_tag(:username, user.username), user_url(user.username))
+        %></td>
+
+        <td class="signed_in-cell" style="horizontal-alginment:center;"><%=
+          if user.lab_sessions.present?
+            singout_time = user.lab_sessions.last.sign_out_time
+            singout_time.strftime("%B %d, %Y")
+          end
+        %></td>
+
+        <td class="location-cell"><%=
+          user.location
+        %></td>
+
+        <td class="certifications-cell"><%
+          if user.certifications.first.present?
```

Comment:
Why do you need this check? If there aren't any certifications you won't loop over anything